components/formats-bsd/src/loci/formats/in/TileJPEGReader.java

```diff
@@ -58,8 +58,7 @@
 
   // -- Fields --
 
-  private JPEGTileDecoder decoder;
-  private JPEGTurboService service;
+  private transient JPEGTurboService service;
```

Comment:
Does this block merging, or is it acceptable given that this is meant for a minor release?